package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "input-plugin-url",
-  "version": "0.0.1",
+  "keywords": "punchcard-cms",
```

Comment:
Can you please add a `"version": "0.0.0"`? That way we can import non-production branches when needed.